Makefile

```diff
@@ -14,11 +14,16 @@ unit-test:
 unit-test-coverage:
 	nyc --reporter=$(if $(CIRCLECI),lcovonly,lcov) make unit-test
 
+integration:
```

Comment:
**question:** I'm not sure _exactly_ how this works, but n-gage already has an `integration-test` target which we could possibly use rather than writing our own. Tbh I'm not sure why this repo defines it's own `unit-test` target either 🤔  is it possible we can lean on n-gage?

I don't think this should block merging, I don't care that much, just wondering if we can save a few lines of code here.

Reply:
Oops no it doesn't! That's the integration tests for n-gage itself 🤦  ignore my comment entirely